open_instruct/utils.py

```diff
@@ -152,9 +152,10 @@ def get_datasets(
     Loads and mixes datasets according to proportions specified in `dataset_mixer`.
 
     Args:
-        dataset_mixer (`dict`):
+        dataset_mixer (`list` or `dict`):
             Dictionary containing the dataset names and their training proportions.
             By default, all test proportions are 1.
+            If a list is passed in, it will be converted to a dictionary.
```

Comment:
Can you give a little example of list format? I'm guess its `key1 value1 key2 value2...`?

Reply:
done!